src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml.cs

```diff
@@ -80,16 +77,13 @@ private async Task Initialize()
                 // Disable the UI.
                 FilterOptions.IsVisible = false;
 
-                // Create and load a service geodatabase that matches utility network.
-                ServiceGeodatabase serviceGeodatabase = await ServiceGeodatabase.CreateAsync(new Uri(FeatureServiceUrl));
-
                 // Create a map with layers in this utility network.
-                MyMapView.Map = new Map(BasemapStyle.ArcGISStreetsNight);
-                MyMapView.Map.OperationalLayers.Add(new FeatureLayer(serviceGeodatabase.GetTable(LineLayerId)));
-                MyMapView.Map.OperationalLayers.Add(new FeatureLayer(serviceGeodatabase.GetTable(DeviceLayerId)));
+                MyMapView.Map = new Map(new Uri("https://sampleserver7.arcgisonline.com/portal/home/item.html?id=f439b4724bb54ac088a2c21eaf70da7b"));
+                await MyMapView.Map.LoadAsync();
 
                 // Create and load the utility network.
```

Comment:
```suggestion
                // Get the utility network from the map.
```

May be this comment instead since we're no longer creating UN?